knexfile.env.js

```diff
@@ -0,0 +1,10 @@
+// Update with your config settings.
```

Comment:
Is this a leftover comment from knex's docs? Or is the implication here that anyone who pulls this change needs to make some personal change to this file?